test/integration/BUILD

```diff
@@ -231,6 +231,20 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "drain_close_integration_test",
+    srcs = [
+        "drain_close_integration_test.cc",
+    ],
+    # REVIEWER: Do I need this tag?
+    # tags = ["fails_on_windows"],
```

Comment:
it looks like CI didn't fail on windows without it.

Reply:
Removed the comments then.